ONSdigital/response-operations-ui
# Update design system to v70

# What and why?
This Pr updated the design system to the latest available version at the time. 
Strong tags in headings are now used to highlight so these have been removed from the code to prevent any unintended Ui changes. 
# How to test?
Run the app and check functionality and UI, Pagination headings should display the same as before. 
# Jira
https://jira.ons.gov.uk/browse/RAS-1094
